pkg/controller/operands/ssp_test.go

```diff
@@ -79,13 +79,14 @@ var _ = Describe("SSP Operands", func() {
 		})
 
 		It("should reconcile to default", func() {
+			cTNamespace := "nonDefault"
+			hco.Spec.CommonTemplatesNamespace = &cTNamespace
 			expectedResource := NewSSP(hco)
 			existingResource := expectedResource.DeepCopy()
 			existingResource.ObjectMeta.SelfLink = fmt.Sprintf("/apis/v1/namespaces/%s/dummies/%s", existingResource.Namespace, existingResource.Name)
 
 			replicas := int32(defaultTemplateValidatorReplicas * 2) // non-default value
 			existingResource.Spec.TemplateValidator.Replicas = &replicas
-			existingResource.Spec.CommonTemplates.Namespace = "foobar"
 			existingResource.Spec.NodeLabeller.Placement = &lifecycleapi.NodePlacement{
 				NodeSelector: map[string]string{"foo": "bar"},
 			}
```

Comment:
This is a good test, but please also add test/test case in pkg/controller/hyperconverged/hyperconverged_controller_test.go